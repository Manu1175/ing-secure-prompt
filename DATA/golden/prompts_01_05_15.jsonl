{"id": "prompts_01_05_15:0", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_05_15.xlsx", "modality": "text", "c_level": null, "raw_text": "What insights can be drawn from the Annual Report published in 2024? Access it here: https://assets.ing.com/m/6e8ace8ade094690/original/Annual-report-2024.pdf", "expected_scrub": "What insights can be drawn from the <DOCUMENT_TYPE> published in <YEAR>? Access it here: <DOCUMENT_LINK>", "entities": null, "notes": null}
{"id": "prompts_01_05_15:1", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_05_15.xlsx", "modality": "text", "c_level": null, "raw_text": "I’m analyzing ING’s performance. What does the Full Year Results from 2024 reveal? Here's the link: https://newsroom.ing.be/ing-continues-to-deliver-strong-growth-in-2024-reinforcing-its-commitment-to-customers-and-society", "expected_scrub": "I’m analyzing ING’s performance. What does the <DOCUMENT_TYPE> from <YEAR> reveal? Here's the link: <DOCUMENT_LINK>", "entities": null, "notes": null}
{"id": "prompts_01_05_15:2", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_05_15.xlsx", "modality": "text", "c_level": null, "raw_text": "Can you summarize the Half Year Results for 2024? I found it here: https://newsroom.ing.be/ing-belgium-builds-on-strong-results-with-608-million-profit-after-first-half-of-2024", "expected_scrub": "Can you summarize the <DOCUMENT_TYPE> for <YEAR>? I found it here: <DOCUMENT_LINK>", "entities": null, "notes": null}
{"id": "prompts_01_05_15:3", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_05_15.xlsx", "modality": "text", "c_level": null, "raw_text": "What kind of data is included in the Pillar 3 Disclosures Q4 2024? Link: https://assets.ing.com/m/5f710bfeff3364d0/original/ING-BE-Pillar-3-Disclosures-Q4-2024.xlsx", "expected_scrub": "What kind of data is included in the <DOCUMENT_TYPE> <YEAR>? Link: <DOCUMENT_LINK>", "entities": null, "notes": null}
{"id": "prompts_01_05_15:4", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_05_15.xlsx", "modality": "text", "c_level": null, "raw_text": "I need help interpreting the Pillar 3 Additional Disclosure from 2024. Here's the document: https://assets.ing.com/m/42e3b0d86ac55855/original/ING-Belgium-Additional-Pillar-III-Report-2024.pdf", "expected_scrub": "I need help interpreting the <DOCUMENT_TYPE> from <YEAR>. Here's the document: <DOCUMENT_LINK>", "entities": null, "notes": null}
{"id": "prompts_01_05_15:5", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_05_15.xlsx", "modality": "text", "c_level": null, "raw_text": "What’s the status of the 'Risk Governance Framework' guideline? I believe it was issued by the National Regulator.", "expected_scrub": "What’s the status of the <GUIDELINE_NAME> guideline? I believe it was issued by the <ISSUING_BODY>.", "entities": null, "notes": null}
{"id": "prompts_01_05_15:6", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_05_15.xlsx", "modality": "text", "c_level": null, "raw_text": "I’m reviewing 'AML Screening Guidelines'. Can you tell me who issued it and whether it’s still active?", "expected_scrub": "I’m reviewing <GUIDELINE_NAME>. Can you tell me who issued it and whether it’s still active?", "entities": null, "notes": null}
{"id": "prompts_01_05_15:7", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_05_15.xlsx", "modality": "text", "c_level": null, "raw_text": "Is 'Digital Onboarding Standards' still considered advisory? Who published it?", "expected_scrub": "Is <GUIDELINE_NAME> still considered <STATUS>? Who published it?", "entities": null, "notes": null}
{"id": "prompts_01_05_15:8", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_05_15.xlsx", "modality": "text", "c_level": null, "raw_text": "I came across 'Sustainability Reporting Protocol'. What’s its category and current status?", "expected_scrub": "I came across <GUIDELINE_NAME>. What’s its <CATEGORY> and current <STATUS>?", "entities": null, "notes": null}
{"id": "prompts_01_05_15:9", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_05_15.xlsx", "modality": "text", "c_level": null, "raw_text": "Can you confirm the status of 'Cybersecurity Controls' issued by the Basel Committee?", "expected_scrub": "Can you confirm the status of <GUIDELINE_NAME> issued by <ISSUING_BODY>?", "entities": null, "notes": null}
{"id": "prompts_01_05_15:10", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_05_15.xlsx", "modality": "text", "c_level": null, "raw_text": "Did Emily Davis complete a wire transfer of 15660.4 EUR to Supplier 764 on 45845?", "expected_scrub": "Did <PAYER_NAME> complete a wire transfer of <AMOUNT> <CURRENCY> to <PAYEE_NAME> on <DATE>?", "entities": null, "notes": null}
{"id": "prompts_01_05_15:11", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_05_15.xlsx", "modality": "text", "c_level": null, "raw_text": "What’s the status of Tom Taylor’s cheque to Anna Davis for 3402.37 USD on 45791?", "expected_scrub": "What’s the status of <PAYER_NAME>’s cheque to <PAYEE_NAME> for <AMOUNT> <CURRENCY> on <DATE>?", "entities": null, "notes": null}
{"id": "prompts_01_05_15:12", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_05_15.xlsx", "modality": "text", "c_level": null, "raw_text": "I need confirmation: did Chris Smith’s SEPA Credit to Tom Davis for 18648.23 GBP on 45827 fail?", "expected_scrub": "I need confirmation: did <PAYER_NAME>’s SEPA Credit to <PAYEE_NAME> for <AMOUNT> <CURRENCY> on <DATE> fail?", "entities": null, "notes": null}
{"id": "prompts_01_05_15:13", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_05_15.xlsx", "modality": "text", "c_level": null, "raw_text": "Was Laura Taylor’s direct debit to Supplier 905 for 141.99 GBP on 45753 successful?", "expected_scrub": "Was <PAYER_NAME>’s direct debit to <PAYEE_NAME> for <AMOUNT> <CURRENCY> on <DATE> successful?", "entities": null, "notes": null}
{"id": "prompts_01_05_15:14", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_05_15.xlsx", "modality": "text", "c_level": null, "raw_text": "Can you confirm if Jane Williams cancelled her standing order to Supplier 896 for 9153.49 GBP on 45376?", "expected_scrub": "Can you confirm if <PAYER_NAME> cancelled her standing order to <PAYEE_NAME> for <AMOUNT> <CURRENCY> on <DATE>?", "entities": null, "notes": null}
{"id": "prompts_01_05_15:15", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_05_15.xlsx", "modality": "text", "c_level": null, "raw_text": "Did Robert Brown’s cheque to Katie Williams for 19478.95 GBP on 45329 go through?", "expected_scrub": "Did <PAYER_NAME>’s cheque to <PAYEE_NAME> for <AMOUNT> <CURRENCY> on <DATE> go through?", "entities": null, "notes": null}
{"id": "prompts_01_05_15:16", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_05_15.xlsx", "modality": "text", "c_level": null, "raw_text": "What’s the current status of Katie Brown’s wire transfer to Supplier 660 for 6014.97 EUR on 45637?", "expected_scrub": "What’s the current status of <PAYER_NAME>’s wire transfer to <PAYEE_NAME> for <AMOUNT> <CURRENCY> on <DATE>?", "entities": null, "notes": null}
{"id": "prompts_01_05_15:17", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_05_15.xlsx", "modality": "text", "c_level": null, "raw_text": "Was the cheque from Emily Williams to Supplier 986 for 15311.43 USD on 45585 cancelled?", "expected_scrub": "Was the cheque from <PAYER_NAME> to <PAYEE_NAME> for <AMOUNT> <CURRENCY> on <DATE> cancelled?", "entities": null, "notes": null}
{"id": "prompts_01_05_15:18", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_05_15.xlsx", "modality": "text", "c_level": null, "raw_text": "Can you confirm if Tom Walker’s direct debit to Supplier 763 for 11428.94 GBP on 45378 was completed?", "expected_scrub": "Can you confirm if <PAYER_NAME>’s direct debit to <PAYEE_NAME> for <AMOUNT> <CURRENCY> on <DATE> was completed?", "entities": null, "notes": null}
{"id": "prompts_01_05_15:19", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_05_15.xlsx", "modality": "text", "c_level": null, "raw_text": "Did Chris Taylor successfully complete a direct debit to John Miller for 10523.35 GBP on 45552?", "expected_scrub": "Did <PAYER_NAME> successfully complete a direct debit to <PAYEE_NAME> for <AMOUNT> <CURRENCY> on <DATE>?", "entities": null, "notes": null}
{"id": "prompts_01_05_15:0", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_05_15.xlsx", "modality": "text", "c_level": null, "raw_text": "What insights can be drawn from the Annual Report published in 2024? Access it here: https://assets.ing.com/m/6e8ace8ade094690/original/Annual-report-2024.pdf", "expected_scrub": "What insights can be drawn from the <DOCUMENT_TYPE> published in <YEAR>? Access it here: <DOCUMENT_LINK>", "entities": null, "notes": null}
{"id": "prompts_01_05_15:1", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_05_15.xlsx", "modality": "text", "c_level": null, "raw_text": "I’m analyzing ING’s performance. What does the Full Year Results from 2024 reveal? Here's the link: https://newsroom.ing.be/ing-continues-to-deliver-strong-growth-in-2024-reinforcing-its-commitment-to-customers-and-society", "expected_scrub": "I’m analyzing ING’s performance. What does the <DOCUMENT_TYPE> from <YEAR> reveal? Here's the link: <DOCUMENT_LINK>", "entities": null, "notes": null}
{"id": "prompts_01_05_15:2", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_05_15.xlsx", "modality": "text", "c_level": null, "raw_text": "Can you summarize the Half Year Results for 2024? I found it here: https://newsroom.ing.be/ing-belgium-builds-on-strong-results-with-608-million-profit-after-first-half-of-2024", "expected_scrub": "Can you summarize the <DOCUMENT_TYPE> for <YEAR>? I found it here: <DOCUMENT_LINK>", "entities": null, "notes": null}
{"id": "prompts_01_05_15:3", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_05_15.xlsx", "modality": "text", "c_level": null, "raw_text": "What kind of data is included in the Pillar 3 Disclosures Q4 2024? Link: https://assets.ing.com/m/5f710bfeff3364d0/original/ING-BE-Pillar-3-Disclosures-Q4-2024.xlsx", "expected_scrub": "What kind of data is included in the <DOCUMENT_TYPE> <YEAR>? Link: <DOCUMENT_LINK>", "entities": null, "notes": null}
{"id": "prompts_01_05_15:4", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_05_15.xlsx", "modality": "text", "c_level": null, "raw_text": "I need help interpreting the Pillar 3 Additional Disclosure from 2024. Here's the document: https://assets.ing.com/m/42e3b0d86ac55855/original/ING-Belgium-Additional-Pillar-III-Report-2024.pdf", "expected_scrub": "I need help interpreting the <DOCUMENT_TYPE> from <YEAR>. Here's the document: <DOCUMENT_LINK>", "entities": null, "notes": null}
{"id": "prompts_01_05_15:5", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_05_15.xlsx", "modality": "text", "c_level": null, "raw_text": "What’s the status of the 'Risk Governance Framework' guideline? I believe it was issued by the National Regulator.", "expected_scrub": "What’s the status of the <GUIDELINE_NAME> guideline? I believe it was issued by the <ISSUING_BODY>.", "entities": null, "notes": null}
{"id": "prompts_01_05_15:6", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_05_15.xlsx", "modality": "text", "c_level": null, "raw_text": "I’m reviewing 'AML Screening Guidelines'. Can you tell me who issued it and whether it’s still active?", "expected_scrub": "I’m reviewing <GUIDELINE_NAME>. Can you tell me who issued it and whether it’s still active?", "entities": null, "notes": null}
{"id": "prompts_01_05_15:7", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_05_15.xlsx", "modality": "text", "c_level": null, "raw_text": "Is 'Digital Onboarding Standards' still considered advisory? Who published it?", "expected_scrub": "Is <GUIDELINE_NAME> still considered <STATUS>? Who published it?", "entities": null, "notes": null}
{"id": "prompts_01_05_15:8", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_05_15.xlsx", "modality": "text", "c_level": null, "raw_text": "I came across 'Sustainability Reporting Protocol'. What’s its category and current status?", "expected_scrub": "I came across <GUIDELINE_NAME>. What’s its <CATEGORY> and current <STATUS>?", "entities": null, "notes": null}
{"id": "prompts_01_05_15:9", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_05_15.xlsx", "modality": "text", "c_level": null, "raw_text": "Can you confirm the status of 'Cybersecurity Controls' issued by the Basel Committee?", "expected_scrub": "Can you confirm the status of <GUIDELINE_NAME> issued by <ISSUING_BODY>?", "entities": null, "notes": null}
{"id": "prompts_01_05_15:10", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_05_15.xlsx", "modality": "text", "c_level": null, "raw_text": "Did Emily Davis complete a wire transfer of 15660.4 EUR to Supplier 764 on 45845?", "expected_scrub": "Did <PAYER_NAME> complete a wire transfer of <AMOUNT> <CURRENCY> to <PAYEE_NAME> on <DATE>?", "entities": null, "notes": null}
{"id": "prompts_01_05_15:11", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_05_15.xlsx", "modality": "text", "c_level": null, "raw_text": "What’s the status of Tom Taylor’s cheque to Anna Davis for 3402.37 USD on 45791?", "expected_scrub": "What’s the status of <PAYER_NAME>’s cheque to <PAYEE_NAME> for <AMOUNT> <CURRENCY> on <DATE>?", "entities": null, "notes": null}
{"id": "prompts_01_05_15:12", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_05_15.xlsx", "modality": "text", "c_level": null, "raw_text": "I need confirmation: did Chris Smith’s SEPA Credit to Tom Davis for 18648.23 GBP on 45827 fail?", "expected_scrub": "I need confirmation: did <PAYER_NAME>’s SEPA Credit to <PAYEE_NAME> for <AMOUNT> <CURRENCY> on <DATE> fail?", "entities": null, "notes": null}
{"id": "prompts_01_05_15:13", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_05_15.xlsx", "modality": "text", "c_level": null, "raw_text": "Was Laura Taylor’s direct debit to Supplier 905 for 141.99 GBP on 45753 successful?", "expected_scrub": "Was <PAYER_NAME>’s direct debit to <PAYEE_NAME> for <AMOUNT> <CURRENCY> on <DATE> successful?", "entities": null, "notes": null}
{"id": "prompts_01_05_15:14", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_05_15.xlsx", "modality": "text", "c_level": null, "raw_text": "Can you confirm if Jane Williams cancelled her standing order to Supplier 896 for 9153.49 GBP on 45376?", "expected_scrub": "Can you confirm if <PAYER_NAME> cancelled her standing order to <PAYEE_NAME> for <AMOUNT> <CURRENCY> on <DATE>?", "entities": null, "notes": null}
{"id": "prompts_01_05_15:15", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_05_15.xlsx", "modality": "text", "c_level": null, "raw_text": "Did Robert Brown’s cheque to Katie Williams for 19478.95 GBP on 45329 go through?", "expected_scrub": "Did <PAYER_NAME>’s cheque to <PAYEE_NAME> for <AMOUNT> <CURRENCY> on <DATE> go through?", "entities": null, "notes": null}
{"id": "prompts_01_05_15:16", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_05_15.xlsx", "modality": "text", "c_level": null, "raw_text": "What’s the current status of Katie Brown’s wire transfer to Supplier 660 for 6014.97 EUR on 45637?", "expected_scrub": "What’s the current status of <PAYER_NAME>’s wire transfer to <PAYEE_NAME> for <AMOUNT> <CURRENCY> on <DATE>?", "entities": null, "notes": null}
{"id": "prompts_01_05_15:17", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_05_15.xlsx", "modality": "text", "c_level": null, "raw_text": "Was the cheque from Emily Williams to Supplier 986 for 15311.43 USD on 45585 cancelled?", "expected_scrub": "Was the cheque from <PAYER_NAME> to <PAYEE_NAME> for <AMOUNT> <CURRENCY> on <DATE> cancelled?", "entities": null, "notes": null}
{"id": "prompts_01_05_15:18", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_05_15.xlsx", "modality": "text", "c_level": null, "raw_text": "Can you confirm if Tom Walker’s direct debit to Supplier 763 for 11428.94 GBP on 45378 was completed?", "expected_scrub": "Can you confirm if <PAYER_NAME>’s direct debit to <PAYEE_NAME> for <AMOUNT> <CURRENCY> on <DATE> was completed?", "entities": null, "notes": null}
{"id": "prompts_01_05_15:19", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_05_15.xlsx", "modality": "text", "c_level": null, "raw_text": "Did Chris Taylor successfully complete a direct debit to John Miller for 10523.35 GBP on 45552?", "expected_scrub": "Did <PAYER_NAME> successfully complete a direct debit to <PAYEE_NAME> for <AMOUNT> <CURRENCY> on <DATE>?", "entities": null, "notes": null}
