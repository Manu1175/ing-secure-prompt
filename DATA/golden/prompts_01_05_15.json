[
  {
    "text": "What insights can be drawn from the Annual Report published in 2024? Access it here: https://assets.ing.com/m/6e8ace8ade094690/original/Annual-report-2024.pdf",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "I’m analyzing ING’s performance. What does the Full Year Results from 2024 reveal? Here's the link: https://newsroom.ing.be/ing-continues-to-deliver-strong-growth-in-2024-reinforcing-its-commitment-to-customers-and-society",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Can you summarize the Half Year Results for 2024? I found it here: https://newsroom.ing.be/ing-belgium-builds-on-strong-results-with-608-million-profit-after-first-half-of-2024",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "What kind of data is included in the Pillar 3 Disclosures Q4 2024? Link: https://assets.ing.com/m/5f710bfeff3364d0/original/ING-BE-Pillar-3-Disclosures-Q4-2024.xlsx",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "I need help interpreting the Pillar 3 Additional Disclosure from 2024. Here's the document: https://assets.ing.com/m/42e3b0d86ac55855/original/ING-Belgium-Additional-Pillar-III-Report-2024.pdf",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "What’s the status of the 'Risk Governance Framework' guideline? I believe it was issued by the National Regulator.",
    "labels": []
  },
  {
    "text": "I’m reviewing 'AML Screening Guidelines'. Can you tell me who issued it and whether it’s still active?",
    "labels": []
  },
  {
    "text": "Is 'Digital Onboarding Standards' still considered advisory? Who published it?",
    "labels": []
  },
  {
    "text": "I came across 'Sustainability Reporting Protocol'. What’s its category and current status?",
    "labels": []
  },
  {
    "text": "Can you confirm the status of 'Cybersecurity Controls' issued by the Basel Committee?",
    "labels": []
  },
  {
    "text": "Did Emily Davis complete a wire transfer of 15660.4 EUR to Supplier 764 on 45845?",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "What’s the status of Tom Taylor’s cheque to Anna Davis for 3402.37 USD on 45791?",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "I need confirmation: did Chris Smith’s SEPA Credit to Tom Davis for 18648.23 GBP on 45827 fail?",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Was Laura Taylor’s direct debit to Supplier 905 for 141.99 GBP on 45753 successful?",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Can you confirm if Jane Williams cancelled her standing order to Supplier 896 for 9153.49 GBP on 45376?",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Did Robert Brown’s cheque to Katie Williams for 19478.95 GBP on 45329 go through?",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "What’s the current status of Katie Brown’s wire transfer to Supplier 660 for 6014.97 EUR on 45637?",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Was the cheque from Emily Williams to Supplier 986 for 15311.43 USD on 45585 cancelled?",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Can you confirm if Tom Walker’s direct debit to Supplier 763 for 11428.94 GBP on 45378 was completed?",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Did Chris Taylor successfully complete a direct debit to John Miller for 10523.35 GBP on 45552?",
    "labels": [
      "PHONE"
    ]
  }
]